plaidml/exec/exec.h

```diff
@@ -59,6 +71,12 @@ inline std::vector<std::string> list_devices() {
   return ret;
 }
 
+///
+/// \ingroup exec_functions
+/// Lists the available targets.
+/// Use this list of targets to set the environment variable `PLAIDML_TARGET`.
```

Comment:
This list isn't really related to env vars itself. This list simply represents the targets that are available. I don't think I'd mention anything related to `PLAIDML_TARGET` here, that probably belongs in a usage/configuration section, it's not really part of the API itself.